.devcontainer/features/kubernetes/install.sh

```
@@ -0,0 +1,429 @@
+echo "Installing Kubernetes Development Tools"
+echo "========================================="
+
+# ─────────────────────────────────────────────────────────────────────────────
+# Pre-installed tool detection (base image may already include kubectl/Helm)
+# ─────────────────────────────────────────────────────────────────────────────
+KUBECTL_PREINSTALLED=false
+HELM_PREINSTALLED=false
+
+if command -v kubectl &>/dev/null; then
+  echo -e "${GREEN}✓ kubectl already installed: $(kubectl version --client --short 2>/dev/null || kubectl version --client 2>/dev/null | head -1)${NC}"
+  KUBECTL_PREINSTALLED=true
```

Shellcheck (0.11.0)

[warning] KUBECTL_PREINSTALLED appears unused. Verify use (or export if used externally).

(SC2034)